webapp/src/ts/modules/messages/messages.routes.ts

```diff
@@ -15,6 +15,19 @@ export const routes: Routes = [
         path: '',
         component: MessagesContentComponent
       },
+      {
+        matcher: (url: UrlSegment[]) => {
```

Comment:
I thought of leaving the 3 separated children in the routes file (one place) and having more clarity on the route possibilities for messages. Avoiding one big matcher doing the 3 scenarios or avoiding redirection in the component.
1. `/messages`
2. `/messages/[id]` where we do matcher and redirect to `/messages/contact:[id]`
3. `/messages/[type]:[id]`